### DISCUSSION TITLE
Template type indicator not updating after modal change

### DISCUSSION THREAD

**Discussion by jcogs-design** [ExpressionEngine/ExpressionEngine#2916] [Feature Requests]
**Description of the problem**
In CP Template Group view, changing the type of template using the modal control (click on cog icon) does not update the file type indicator in table upon return (you need to refresh the page to see change).

**How To Reproduce**
1. Open CP / Templates and click on 'cog' icon against a template
2. Change the file type for the template in modal dialog
3. Save changes to return to list view
4. File type does not reflect change made in modal - until you refresh the page

**Error Messages**

**Screenshots / Videos / Template Code**

**Environment Details:**
 - Version: 5.3.2
 - PHP Version 7.3.21
 - MySQL Version 10.3.23-MariaDB
 - OS: macOS 10.14.6
 - Web Server: LiteSpeed V7.7 

**Possible Solution**

**Additional context**

**Comment by intoeetive** [comment]
confirming I see the same in EE6. The template name does not change too

**Comment by jcogs-design** [comment]
Just a note that this issue is still present on EE6.2.1 :(
